02/app.js

```diff
@@ -0,0 +1,71 @@
+const formEl = document.querySelector('form')
+const errors = []
+console.log(errors)
+
+formEl.addEventListener('submit', function (e) {
+	e.preventDefault()
+
+	checkEmail(e)
+	checkPassword(e)
+	acceptRegulations(e)
+	showErrors(e)
+})
+
+function checkEmail(e) {
```

Comment:
Tutaj podobnie jak wcześniej funkcja niby sprawdza email, ale robi po drodze wiele innych rzeczy, czytelniej będzie to rozdzielić:
```
if(isCorrectEmail(e)) { // nowa nazwa funkcji, zamiast checkEmail
  clearErrors(inpEmail)
} else {
  errors.push(inpEmail)
}
```